WcaOnRails/app/models/post.rb

```diff
@@ -26,7 +26,7 @@ def edit_path
     if is_crash_course_post?
       Rails.application.routes.url_helpers.delegate_crash_course_edit_path
     else
-      Rails.application.routes.url_helpers.edit_post_path(slug)
+      Rails.application.routes.url_helpers.edit_post_path(id: slug)
```

Comment:
Hmm... I'm a bit confused why this isn't an issue elsewhere in the site. What about something like this: https://github.com/cubing/worldcubeassociation.org/blob/9fbeae204994ae38512bb2ed5563eac6469182db/WcaOnRails/app/controllers/posts_controller.rb#L31?

```ruby
      redirect_to post_path(@post.slug)
```